h2o-core/src/main/java/water/H2O.java

```diff
@@ -303,6 +303,9 @@ public static void printHelp() {
 
     public boolean useUDP = false;
 
+    /** -nolatest Do not attempt to retrieve latest H2O version from S3 on startup */
+    public boolean noLatest = false;
```

Comment:
Personal comment: `no latest` is confusing - sounds like "do not use latest version". 

What about 'disable_check_for_latest' ?